mlrun/feature_store/steps.py

```diff
@@ -70,6 +70,9 @@ def _do_storey(self, event):
     def _do_spark(self, event):
         raise NotImplementedError
 
+    def validate(self, entities=None):
```

Comment:
It's good that you went for a more generic validation approach, but with the current signature of this method (i.e. getting just the entities), the only validation you'll be able to do would be regarding entities. This works for the `DropFeatures` step, but will not extend once new validations will be needed. If you want this to be a generic validation capability, you should probably pass more information - perhaps getting the full `FeatureSet` object that holds the step.

Reply:
great comment. done!